Gemfile

```diff
@@ -37,6 +37,9 @@ gem 'jbuilder', '~> 2.5'
 gem 'bootsnap', '>= 1.1.0', require: false
 
 gem 'devise', '~> 4.5'
+gem 'devise_token_auth', '~> 1.0'
+gem 'rack-cors', require: 'rack/cors'
```

Comment:
I do not understand why you needed to add this gem? Something is causing you issue with the `protect_from_foregery` config?

Reply:
No, I'm not having issues with protect_from_forgery. Just added the gem as devise token auth suggested to avoid CORS issues if the API is used  from another domain. But currently we really don't need that since we are not creating a front end project to use this API, I just wanted to do it right now and not in the future, but I'll remove that gem. Thanks.